stacks/azure/application/tensorboard/kustomization.yaml

```diff
@@ -0,0 +1,10 @@
+apiVersion: kustomize.config.k8s.io/v1beta1
+kind: Kustomization
+namespace: kubeflow
+resources:
+- ../../../../tensorboard/overlays/istio
```

Comment:
Just a heads up that the TensorBoard-related manifests were removed in https://github.com/kubeflow/manifests/pull/1459, and keeping in references to this will result in errors. So maybe best to remove the `application/tensorboard` directory.

Reply:
Thanks @pvaneck will remove!

@Jeffwan no need for additional Azure reviews -- see my description. These were already reviewed commits against the v1.1 branch and I'm just cherry picking them back into master.